osu.Game.Rulesets.Catch/Skinning/CatchSkinConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Catch.Skinning
+{
+    public enum CatchSkinConfiguration
+    {
+        /// <summary>
+        /// Whether the contents of the catcher plate should be visually flipped when the catcher direction is changed.
+        /// </summary>
+        FlipCatcherPlate
```

Comment:
Still not sure about adding new values to these areas (was intended to only be for actually supported values in osu-stable skin versions) but I don't have a better suggestion for now.